charts/osm/values.yaml

```diff
@@ -106,7 +106,7 @@ OpenServiceMesh:
   # -- Enables HTTPS ingress on the mesh
   useHTTPSIngress: false
   # -- Envoy log level is used to specify the level of logs collected from envoy
-  envoyLogLevel: error
+  envoyLogLevel: debug
```

Comment:
Is this somehow related to the OPA work? What's the reason to change it?

Reply:
+1, this change should be dropped.

Reply:
misspush